remote-admin-create-cache/src/main/java/org/infinispan/tutorial/simple/remote/admin/InfinispanRemoteAdminCache.java

```diff
@@ -0,0 +1,98 @@
+package org.infinispan.tutorial.simple.remote.admin;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.client.hotrod.impl.ConfigurationProperties;
+import org.infinispan.commons.api.CacheContainerAdmin.AdminFlag;
+import org.infinispan.commons.configuration.XMLStringConfiguration;
+
+/**
+ * The Administration simple tutorial to show how create caches with a remote
+ * client.
+ * There are some caches created by using different approach,
+ * most of them are persistent and will survive a restart.
+ * The temporary cache is also replicated across the cluster but lost on a full restart.
+ *
+ * @author <a href="mailto:wfink@redhat.com">Wolf Dieter Fink</a>
+ */
+public class InfinispanRemoteAdminCache {
+    private final RemoteCacheManager manager;
+
+    private InfinispanRemoteAdminCache() {
+        // Create a configuration for a locally-running server
+        ConfigurationBuilder builder = new ConfigurationBuilder();
+        builder.addServer().host("127.0.0.1").port(ConfigurationProperties.DEFAULT_HOTROD_PORT);
+
+        manager = new RemoteCacheManager(builder.build());
+    }
+
+    public static void main(String[] args) throws Exception {
+        InfinispanRemoteAdminCache client = new InfinispanRemoteAdminCache();
+
+        client.createSimpleCache();
+        client.createCacheWithCacheConfiguration();
+        client.createCacheWithXMLConfiguration();
+
+        // Stop the client and release all resources
+        client.stop();
+    }
+
+    private void stop() {
+        manager.stop();
+    }
+
+    /**
+     * For this a configuration is needed within the infinispan.xml file!
+     * Add the following part to the <cache-container> element
+     * <code>
+     *    <distributed-cache-configuration name="MyDistCachecConfig" mode="ASYNC">
+     *      <state-transfer await-initial-transfer="false"/>
+     *     </distributed-cache-configuration>
+     * </code>
+     */
+    private void createCacheWithCacheConfiguration() {
+        manager.administration().getOrCreateCache("CacheWithConfigurationTemplate", "MyDistCachecConfig");
```

Comment:
If the user does not change the configuration in the infinispan.xml in the server, this will fail and all the method will fail.
For me this should display a message in the comment above when you run the main explaining that the template needs to be in the server and maybe providing the xml file and the docker command to pass it when we run the server using docker container or how to passe it with a standalone server

Reply:
I would call this more "create cache with configuration template"

Reply:
Maybe adding a readme for running the server and how to passe this infinispan.xml